tests/system/action/meeting/test_clone.py

```diff
@@ -530,3 +531,71 @@ def test_permissions_missing_payload_committee_permission(self) -> None:
             "Missing CommitteeManagementLevel: can_manage for committee 2",
             response.json["message"],
         )
+
+    def test_clone_with_created_topic_and_agenda_type(self) -> None:
+        self.set_models(self.test_models)
+        response = self.request(
+            "topic.create",
+            {
+                "meeting_id": 1,
+                "title": "test",
+                "agenda_type": AgendaItem.INTERNAL_ITEM,
+                "agenda_duration": 60,
+            },
+        )
+        self.assert_status_code(response, 200)
+        topic_fqid = f'topic/{response.json["results"][0][0]["id"]}'
+        topic = self.assert_model_exists(
+            topic_fqid, {"agenda_type": None, "agenda_duration": None}
```

Comment:
Better test explicitly that the fields are not present. This creates the impression that the fields are present in the model and set to `None`.

Reply:
done